src/components/Splash/Splash.js

```diff
@@ -1,49 +1,113 @@
 import React, { useState } from 'react';
-import { NavLink } from 'react-router-dom';
-import axios from 'axios';
+import { NavLink, useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { loginUser } from '../../Redux/Authenticate/authentication';
 import mainLogo from '../../img/YELLOW-3.png';
 import './Splash.css';
 
 const Splash = () => {
-  const [userName, setUserName] = useState('');
-  const [userPassword, setUserPassword] = useState('');
-  const loginURL = 'http://localhost:3001/users';
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
   const marianaGithub = 'https://github.com/marurevi';
   const andresGithub = 'https://github.com/anagudelogu';
   const axelGithub = 'https://github.com/AxelSoler';
   const tadesseGithub = 'https://github.com/Tadesse-Alemayehu';
```

Comment:
**[OPTIONAL]** Since these don't change over time, we can consider them constants. It would be a great idea to follow the naming conventions for those and write them in uppercase.

Reply:
- Actually, this is changes from Axel. We might have to discuss this alongside at the end.